.github/workflows/ci_action_kinetic.yml

```diff
@@ -0,0 +1,30 @@
+name: CI-Kinetic
+
+on: [push, pull_request]
+
+env:
+  ADDITIONAL_DEBS: "clang-tidy libclang-dev"
+  CATKIN_LINT: true
+  CATKIN_LINT_ARGS: '--strict --ignore env_var'
+  CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
+  ROS_DISTRO: kinetic
+
+jobs:
+  testing-repo:
+    name: "Build + Test with Testing Repo of kinetic (http://packages.ros.org/ros-testing/ubuntu)"
+    runs-on: ubuntu-18.04
```

Comment:
Is this intential? kinetic runs on 16.04 @agutenkunst

Reply:
As far as I remember, the appropriate docker image is selected based on `ROS_DISTRO` - regardless of the host OS.